.eslintrc.json

```diff
@@ -0,0 +1,21 @@
+{
+	"parserOptions": {
+		"ecmaVersion": 6,
+		"sourceType": "module",
+		"ecmaFeatures": {
+			"jsx": true
+		}
+	},
+	"rules": {
+		"indent": ["error", 2, {
+			"VariableDeclarator":{
+				"var": 2,
+				"let": 2,
+				"const": 3
+			}
+		}],
+		"semi": [2, "always"],
+		"quotes": [1, "single"],
+		"no-mixed-spaces-and-tabs": [2, "smart-tabs"]
+	}
```

Comment:
let's add the style integrations - https://eslint.org/docs/rules/#stylistic-issues 
* "single var per line" 
* camelCase naming convention
* comma spacing?, e.g. `var foo = 1, bar = 2;` vs `var foo = 1 ,bar = 2;`

Reply:
@rainrivas we were running in to an issue with camelCasing last night. We only wanted to camel case variables, but constants should be in all CAPS, didnt know how to set up up for constant declarations vs var declarations. Maybe we cna still figure it out

Reply:
@jwu910 I was reading the docs and it said something about it ignoring vars that are all caps and that "_" was an escape character as well. Docs also read, "If ESLint decides that the variable is a constant (all uppercase), then no warning will be thrown."
Maybe I can mess with it later on to see what we can do with it

Reply:
oh interesting @rainrivas  that would probably solve that issue. We can add it in and test. Good to have more minds on it to catch more use cases to address the linting needs.

@AlChiu anything you want to add?

Reply:
Hmmmm.... For me I usually default to a C++ style that I've used for sometime. 

@rainrivas Camel case of variables is a big thing for me, and I adhere to it like a fanatic. Same with all caps for constants. I tend to group my variable declarations so that it is like
```cpp
int a, b, c; // I do this

int a;  // I avoid this as much as I can
int b;
int c;
```

For example: 
```cpp
for (int i = 0; i < n; ++i) {
  j += i;  // Some comment
}
```

Notice that I have put a space between the 'for' and the first parenthesis. The first brace is one space away from the second parenthesis and is on the same line with 'for'. Indents are two spaces. The start of inline comments are always two spaces away from the code. It is also best to keep each line of code to be less than 80 characters, if possible.

@jwu910 @rainrivas @phuchle Not sure if this is what you want.

Reply:
@AlChiu for JS the convention is 
```javascript
var a = 'thing',
b = ' thing2',
c = 'thing3';
```
we wont be declaring empty vars often